src/tomato_bronkhorst/__init__.py

```diff
@@ -0,0 +1,61 @@
+from tomato.driverinterface_1_0 import ModelInterface, Attr, Task
+from propar import PP_TYPE_FLOAT, instrument as Instrument
+from typing import Any
+
+class DriverInterface(ModelInterface):
+    class DeviceManager(ModelInterface.DeviceManager):
+        instrument: Instrument
+
+        def __init__(self, driver: ModelInterface, key: tuple[str, int], **kwargs: dict):
+            super().__init__(driver, key, **kwargs)
+            address, channel = key
+            self.instrument = Instrument(comport = address, address = channel)
+
+        def do_task(self, task: Task, **kwargs):
+            # TODO: implement task loop
+            pass
+
+        def set_attr(self, attr: str, val: Any, **kwargs: dict):
+            # TODO: implement setters
+            pass
+
+        def get_attr(self, attr: str, **kwargs: dict):
+            property_map = {
+                'temperature': {'proc_nr': 33, 'parm_nr': 7, 'parm_type': PP_TYPE_FLOAT},
+                'flow': {'param_id': 205},
+                'pressure': {'proc_nr': 33, 'parm_nr': 5},
+                'max_flow': {'param_id': 128},
+                'flow_unit': {'param_id': 129}
+            }
```

Comment:
@AlexN7-Shepard We could also move the `property_map` out of this class as it only needs to be defined once; then it can be reused for `set_attr` above.